policy/custom/definitions/policyset/LogAnalytics.bicep

```diff
@@ -771,6 +771,48 @@ resource policyset_name 'Microsoft.Authorization/policySetDefinitions@2020-03-01
           }
         }
       }
+      {
```

Comment:
You will need to remove the built-in policy definition for Recovery Service vault.

Block to remove since it's being replaced with custom policies.

```
      {
        groupNames: [
          'BUILTIN'
        ]
        policyDefinitionId: '/providers/Microsoft.Authorization/policyDefinitions/c717fb0c-d118-4c43-ab3d-ece30ac81fb3'
        policyDefinitionReferenceId: toLower(replace('Deploy Diagnostic Settings for Recovery Services Vault to Log Analytics workspace for resource specific categories', ' ', '-'))
        parameters: {
          logAnalytics: {
            value: '[parameters(\'logAnalytics\')]'
          }
        }
      }
```